calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -52,6 +52,16 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
 
+:::note
```

Comment:
It sounds like this should be a warning, not a note. It is concerning we allow users to do this at all.  But if this is an interim fix, suggest changing this to a warning. If user does this in error, it would be good to add what the symptoms are in troubleshooting so users can self-diagnose.

Reply:
Thanks Barbara. For now, I changed note to warning. I'll check with Sara/Seth about the symptoms.